### PR TITLE
update all patacrep.com to patacrep.fr

### PR DESCRIPTION
**Pull request by atsju**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -74,7 +74,7 @@ The full documentation is hosted by readthedoc, here : http://patacrep.readthedo
 Contact & Forums
 ----------------
 
-* http://www.patacrep.com/forum
+* http://www.patacrep.fr/forum
```
Comment (sgelis) [inline README.rst:77]: URL seems dead, might as well remove it?

**Review by sgelis — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (atsju) [reply README.rst:77]: possibly yes. Or shall we fix the forum ? (or remove forum from website). 
Who manages website ?

Comment (oliverpool) [reply README.rst:77]: @crep4ever is the owner of the website.

The forum link can probably be replaced with a link to github issues.

Comment (atsju) [inline README.rst:77]:
```suggestion
* https://github.com/patacrep/patacrep/issues
```

**Review by oliverpool — APPROVED** [review]

**Pull request merged**